appium/webdriver/extensions/ime.py

```diff
@@ -34,7 +34,7 @@ def available_ime_engines(self) -> List[str]:
         Returns:
             :obj:`list` of :obj:`str`: The available input methods for an Android device
         """
-        return self.execute(Command.GET_AVAILABLE_IME_ENGINES, {})['value']
+        return self.execute(Command.GET_AVAILABLE_IME_ENGINES, {}).get('value', [])
```

Comment:
I would rather keep it as it was. A user might get an incorrect response if the value is missing instead of an error, which is worse

Reply:
same below

Reply:
okay. this was just to fix lint in VSCode